src/QMCDrivers/DMC/WalkerControlMPI.cpp

```diff
@@ -513,26 +519,12 @@ void WalkerControlMPI::swapWalkersSimple(MCPopulation& pop,
   // Sort only on the number of copies
   std::sort(sorted_good_walkers.begin(), sorted_good_walkers.end(), [](auto& a, auto& b) { return a.first < b.first; });
 
-  //useful counts
-  int nswap       = plus.size();
-  int local_sends = 0;
-  int local_recvs = 0;
-
-  for (int ic = 0; ic < nswap; ic++)
-  {
-    if (minus[ic] == MyContext)
-      ++local_sends;
-    else if (plus[ic] == MyContext)
-      ++local_recvs;
-  }
+  int nswap = plus.size();
 
   std::vector<WalkerMessage> send_message_list;
   // overallocated by number of duplicate messages but message isn't big.
-  send_message_list.reserve(local_sends);
```

Comment:
reserve() calls are removed but the comment line was not. Pelase also explain why not reserving any more.